app/facades/footprint_facade.rb

```diff
@@ -8,5 +8,20 @@ def create_footprint(car, mileage)
         offset_cost_currency: response[:cloverly_offset_cost][:currency]
       )
     end
+
+    def aggregate_footprint_for_month(user_id, month, year)
```

Comment:
I don't think we want this to be in our footprint_facade, but rather our footprint model. That means we will need to revise the method itself.